docs/Learning-Environment-Create-New.md

```diff
@@ -404,27 +404,28 @@ and include the following hyperparameter values:
 ```yml
 behaviors:
   RollerBall:
-    trainer: ppo
-    batch_size: 10
-    beta: 5.0e-3
-    buffer_size: 100
-    epsilon: 0.2
-    hidden_units: 128
-    lambd: 0.95
-    learning_rate: 3.0e-4
-    learning_rate_schedule: linear
-    max_steps: 5.0e4
-    memory_size: 128
-    normalize: false
-    num_epoch: 3
-    num_layers: 2
+    trainer_type: ppo
+    hyperparameters:
+      batch_size: 10
+      buffer_size: 100
+      learning_rate: 0.0003
+      beta: 0.0005
```

Comment:
I find it easier to read this way. Any reason this changed ?
```suggestion
      learning_rate: 3.0e-4
      beta: 5.0e-3
```

Reply:
It's not the standard YAML conversion, I think scientific notation isn't used until it's `e-5` or more. But it will still load so I'm not opposed to changing it.